internal/common/enums.rs

```diff
@@ -431,6 +431,18 @@ macro_rules! for_each_enums {
                 /// Scrollbar always visible
                 AlwaysOn,
             }
+
+            // This enum describes the close behaviour of [`PopupWindow`](elements.md#popupwindow)
+            enum ClosePolicy {
+                /// Closes the `PopupWindow` when user clicks.
+                OnClick,
+
+                /// Closed the `PopupWindow` when user clicks outside of the popup.
+                OnClickOutside,
```

Comment:
Please add a `Close` prefix. The variant should emphasise what happens when the policy is active. That means a verb is needed.

(strong feeling here again ;-)